packages/app/app/containers/SoundContainer/sleeptimer.js

```diff
@@ -0,0 +1,32 @@
+let props;
+let sleeptimerSleepDate;
+let sleeptimerTimeToSleep=0;
+
+export function putToSleep (callProps) {
```

Comment:
The arguments for this function should be typed, because it's hard to infer what exactly is needed. Either do not pass props, and pass each required argument separately, or pass an object with the arguments.